rs/registry/canister/src/mutations/do_create_subnet.rs

```diff
@@ -299,6 +305,11 @@ pub struct CreateSubnetPayload {
     pub gossip_retransmission_request_ms: u32,
 }
 
+#[derive(Clone, Eq, PartialEq, Debug, Default, CandidType, Deserialize, Serialize)]
+pub struct NewSubnet {
```

Comment:
I'm guessing we already have a SubnetId candid type.  You could make this CreateSubnetResponse, so that if other data has to be added, it can be.  I guess it technically doesn't matter what it's called, but it might be a nice clue for future readers.